docs/codelabs/building-beautiful-sites/docsite/1-setup.md

```diff
@@ -38,4 +38,4 @@ Voilà! Shrine is running in your browser. You can scroll through the page to se
 ![Shrine](img/03-starter.png)
 {: .codelab-img}
 
-[Next step: Explore the starter code](/components/web/codelabs/beautiful-sites-simplified/2-explore/)
+[Next step: Explore the starter code](./2-explore.md)
```

Comment:
Are these changes going to have an adverse effect anywhere else? I'm not sure how the codelab has been published in the past.

(Also, are these files even exposed on material.io? i.e. do these even need to be processed?)

Reply:
Yes. First to your question, codelab is new to our repository, the content wasn't host by us in the past. 
The links here originally are not working on Github but only on catalog site, which is a broken link. Hope that make things clear?